### PR TITLE
BC-2307 set timeout message

### PR DESCRIPTION
**Pull request by bischofmax**
# Short Description

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/store/copy.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ export default class CopyModule extends VuexModule {
 		} catch (error: any) {
 			this.setLoading(false);
 			this.setBusinessError({
-				statusCode: error?.response?.status,
+				statusCode: `${error?.response?.status}`,
```
Comment (SevenWaysDP) [inline src/store/copy.ts:83]: Why do you convert it to string?

File: src/components/copy-result-modal/CopyResultModal.vue [hunk 2]
```diff
@@ -124,8 +141,18 @@ export default {
 		items() {
 			return this.copyResultItems;
 		},
+		hasTimeoutError() {
+			return (
+				this.copyResultError !== undefined &&
+				this.copyResultError.statusCode === "504"
```
Comment (SevenWaysDP) [inline src/components/copy-result-modal/CopyResultModal.vue:147]: You can check it with number?

**Review by SevenWaysDP — CHANGES_REQUESTED** [review]

File: src/store/copy.ts (hunk 1 above)
Comment (BjoernDataport) [reply src/store/copy.ts:83]: ``export type BusinessError = {
	statusCode: string;
	message: string;
	error?: object;
};``

File: src/components/copy-result-modal/CopyResultModal.vue (hunk 2 above)
Comment (BjoernDataport) [reply src/components/copy-result-modal/CopyResultModal.vue:147]: ...the same as above

**Review by SevenWaysDP — APPROVED** [review]

**Pull request merged**